docs/cloud/intro.md

```diff
@@ -0,0 +1,48 @@
+# Feldera Cloud
+
+Feldera Cloud is the enterprise offering of Feldera. It is designed such that
+you can bring your own cloud, on which you deploy Feldera.
+The documentation we provide here is publicly available, both to guide
+our existing customers and to provide insight to interested parties.
+A portion of the Feldera assets (such as container images) required to
+actually perform the steps in this documentation are not publicly
+available, and are only accessible with a (trial) license key.
+
+Please contact us at `learnmore@feldera.com` if you are interested to learn
+more or would like a trial. We'd be happy to get you started!
+
+:::tip
+Don't forget to first try out the [Docker demo container](../intro), which requires
+no (trial) license key. The Docker demo already showcases many of Feldera's features.
+This documentation is about setting up a distributed cloud deployment.
+:::
+
+## Setup
+
+1. **[Load assets](assets):** using your (trial) license key, load the required assets
+   into your cloud environment such that they are available for deployment.
+
+2. **[Deployment](deployment):** start a Kubernetes cluster on which Feldera Cloud 
+   is deployed.
+
+## Usage
+
+Once you have completed the setup, you can get started with making use of all
```

Comment:
> Once you have completed the setup, you can get started with making use of all that Feldera has to offer.

Once you have completed the setup, you can get started with using Feldera Cloud.